src/config/mod.rs

```diff
@@ -42,14 +45,25 @@ impl Default for Config {
 }
 
 pub fn parse_args() -> TofndResult<Config> {
+    // need to use let to avoid dropping temporary value
+    let gg20_port = &DEFAULT_GG20_PORT.to_string();
+    let multisig_port = &DEFAULT_GG20_PORT.to_string();
+
     let app = App::new("tofnd")
         .about("A threshold signature scheme daemon")
         .arg(
-            Arg::with_name("port")
-                .long("port")
-                .short("p")
+            Arg::with_name("gg20-port")
+                .long("gg20-port")
+                .short("g")
+                .required(false)
+                .default_value(gg20_port),
+        )
+        .arg(
+            Arg::with_name("multisig-port")
+                .long("multisig-port")
+                .short("s")
```

Comment:
I'm also fine with not having single char aliases for the port arguments in favour of readability.

Reply:
no need for `multisig-port` anymore